.github/workflows/build.yaml

```
@@ -72,4 +87,77 @@ jobs:
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: bin-${{ matrix.os }}
+          path: ./*.tar.bz2
+
+  test_vad:
+    needs: build
+    name: vad ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix: ${{ fromJson(needs.build.strategy.matrix) }}
```

actionlint (1.7.7)

property "strategy" is not defined in object type {outputs: {}; result: string}

(expression)